marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/flow.sjs

```diff
@@ -243,6 +243,8 @@ class Flow {
         ].reduce((previousValue, currentValue) => (previousValue || []).concat((currentValue || [])))
           // filter out any null/empty collections that may exist
           .filter((col) => !!col);
+        
+        const targetCollectionsAdditivity = options.targetCollectionsAdditivity && String(options.targetCollectionsAdditivity) == "true";
```

Comment:
We will have to check for the property in 'combinedOptions' not just 'options'

Reply:
Fixed